test/Unit/Api/Resources/SkuTest.php

```diff
@@ -25,24 +25,4 @@ public function testUpdatePassesThroughToConnection()
 
         $sku->update();
     }
-
-    public function testOptionsPassesThroughToConnection()
```

Comment:
yes but you still have functionality so should have a test of some kind don't you think?